docs/src/app/components/AppBar/README.md

```diff
@@ -0,0 +1,7 @@
+## AppBar
+
+App bars are a collection of components placed as a static header for an application.
+It is used for navigation, search branding, and actions.
+An app bar is also referred to as the primary toolbar or action bar for Android.
+
+### Examples
```

Comment:
I think the best practice would be to add a `HeaderSection` component that takes a title and a description markdown text and renders the header section. just like the `PropTypeDescription`,